packages/access/access-control-macros/src/lib.rs

```diff
@@ -195,3 +221,112 @@ fn match_path_is_address(ty: &Type, param_name: &Ident, is_ref: bool) -> bool {
 fn panic_type(param_name: &Ident) -> ! {
     panic!("Parameter `{param_name}` must be of type `Address` or `&Address`");
 }
+
+/// A procedural macro that ensures the parameter has any of the specified
+/// roles.
+///
+/// # Security Warning
+///
+/// **IMPORTANT**: This macro does NOT enforce authorization. This is a
+/// deliberate choice, since in Stellar contracts, duplicate `require_auth()`
+/// calls for the same account within the same call stack panics. This macro is
+/// designed for use cases where you want to further limit a function that
+/// already has `require_auth()` in it with access control roles. If you also
+/// need `require_auth()` provided by the macro, please use `#[only_any_role]`
+/// instead.
```

Comment:
```suggestion
/// already has `require_auth()` in it with access control roles. If you also
/// need `require_auth()`, please use `#[only_any_role]`  instead.
```
I understand as if this macro provides the `require_auth()`, which not the case.

Reply:
Quoting: 
```
/// This macro is
/// designed for use cases where you want to further limit a function that
/// already has `require_auth()` in it with access control roles.
```

`a function that already has `require_auth()` in it`

I don't think there is ambiguity?